src/mod/applications/mod_dptools/mod_dptools.c

```diff
@@ -3185,6 +3185,7 @@ SWITCH_STANDARD_APP(endless_playback_function)
 		switch_channel_set_variable(channel, SWITCH_CURRENT_APPLICATION_RESPONSE_VARIABLE, "FILE NOT FOUND");
 		break;
 	default:
+		switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_ERROR, "Could not play the media correctly. error: %s\n", error);
```

Comment:
Take note that we have several playback method:
playback
loop_playback
endless_playback

^ you only add the log on the endless_playback.

Reply:
you also need to print the value of status for additional information

Reply:
Np, fixed it. :)